src/command_modules/azure-cli-interactive/tests/test_completion.py

```diff
@@ -137,8 +136,7 @@ def init4(self):
             "create --funtimes": "There is no work life balance, it's just your life"
         }
         same_param_doubles = {
-            "-f": "--funtimes",
-            "--funtimes": '-f'
+            "create": {"-f": "--funtimes", "--funtimes": '-f'}
```

Comment:
That's correct, your dictionary would look like that. This would not work if there were more than two aliases, which commands do so I can work that in.